app/assets/javascripts/application.js

```diff
@@ -16,3 +16,6 @@
 //= require activestorage
 //= require turbolinks
```

Comment:
turbolinks này có tác dụng gì nhể. :v

Reply:
turbolink có tác dụng tăng hiệu năng trang web
khi click vào thẻ bất kì trên trang thay vì redirect tới một trang khác thì Turbolinks sẽ tạo một Ajax request, sau khi nhận được kết quả trả về sẽ thay thế toàn bộ nội dung của thẻ body bằng nội dung của thẻ body trong kết quả trả về